### PR TITLE
Option to add custom headers in jirashell.py script 

### PR DESCRIPTION
**Pull request by chillipeper**
Hey,

I work with a private instance of Jira Server, and found my self in the necessity to run jirashell.py script with custom headers. So added an option to do this to the original jirashell.py script

Hope this helps

### CODE REVIEW

**Comment by dbaxa** [comment]
@chillipeper hi the travic ci build is failing please take a look and update this pr to fix the flake8 issue that is causing the build failure.

```

./jira/jirashell.py:34:1: H405  multi line docstring summary not separated with an empty line
./jira/jirashell.py:34:25: H403  multi line docstrings should end on a new line
1       H403  multi line docstrings should end on a new line
1       H405  multi line docstring summary not separated with an empty line
ERROR: InvocationError: '/home/travis/build/pycontribs/jira/.tox/py27-linux/bin/python -m flake8'
```

**Comment by chillipeper** [comment]
@dbaxa  fixed

**Comment by asqui** [comment]
Out of curiosity, what custom headers did you find that you need to set in order to connect to your JIRA instance?

**Comment by chillipeper** [comment]
@asqui: Samaccountname, OpenID-Extras

File: jira/jirashell.py [hunk 1]
```diff
@@ -29,6 +29,17 @@
     os.path.expanduser('~'), '.jira-python', 'jirashell.ini')
 
 
+class StoreDictKeyPair(argparse.Action):
+    '''Stores dictionalry key pairs with format key1=value1,key2=value2 into value namespace'''
```
Comment (ssbarnea) [inline jira/jirashell.py]: dictionalry seems as a typo.

**Pull request closed without merging**